pkg/vtpm/src/server.cpp

```diff
@@ -56,35 +212,41 @@ int readMessage(int sock, google::protobuf::uint32 size)
     CodedStrmInput.PopLimit(msgLimit);
 
     cout << "Received command is " << request.command() << std::endl;
+    if (sanitizeCmdRequest(sock, request, response) != 0) {
+         return -1;
+    }
     for (int i=0; i < request.inputfiles_size(); i++) {
         const eve_tools::File& file = request.inputfiles(i);
         cout << "Processing file: " << file.name() << std::endl;
         ofstream input_file;
         input_file.open(file.name(), ios::out|ios::binary);
         if (!input_file) {
-            cout << "Unable to open test file for writing" << std::endl;
+            cout << "Unable to open input file for writing" << std::endl;
             return -1;
```

Comment:
Send error to caller as the santize function does.

Reply:
fixed it in the latest diff.